templates/repo/file_info.tmpl

```diff
@@ -0,0 +1,29 @@
+<div class="file-info text grey normal mono">
+	{{if .FileIsSymlink}}
+		<div class="file-info-entry">
+			{{.locale.Tr "repo.symbolic_link"}}
+		</div>
+	{{end}}
+	{{if .NumLinesSet}}
```

Comment:
Isn't that attribute a bit overkill, if we could use
```suggestion
	{{if .NumLines}}
```
just as well?

Reply:
`.NumLinesSet` is necessary so we can show 0 `.NumLines`.

Reply:
```suggestion
	{{if .NumLinesSet}}{{/* Needed to show 0 line changes */}}
```